core/app/models/concerns/spree/user_address_book.rb

```diff
@@ -92,7 +93,7 @@ def persist_order_address(order)
     # treated as value equality to de-dup among existing Addresses
     # @param default set whether or not this address will show up from
     # #default_address or not
-    def save_in_address_book(address_attributes, default = false)
+    def save_in_address_book(address_attributes, default = false, address_kind = :shipping)
```

Comment:
I changed the name, but in this case I kept the argument as a regular one because the previous argument has a default value, which would not play well with subsequent keyword arguments.